index.html

```diff
@@ -205,19 +198,20 @@ <h3>Locales and Internationalization</h3>
         <p><em>This section defines basic terminology related to internationalization and localization.</em></p>
         
         <p>Users who speak different languages or come from different cultural backgrounds usually require software and services that are adapted to correctly process information using their native languages, writing systems, measurement systems, calendars, and other linguistic rules and cultural conventions.</p>
+<p><a>Language tags</a> can also be used to identify <a>international preferences</a> associated with a given piece of content or user because these preferences are linked to the natural language, regional association, or culture of the end user. Such preferences are applied to processes such as presenting numbers, dates, or times; sorting lists linguistically; providing defaults for items such as the presentation of a calendar, or common units of measurement; selecting between 12- vs. 24-hour time presentation; and many other details that users might find too tedious to set individually. Collectively, an identifier for these preferences is usually called a <a>locale</a>. The extensions to [[BCP47]] that define <a>Unicode locales</a> [[CLDR]] provide the basis for <a>internationalization</a> APIs on the Web, notably the JavaScript language [[ECMASCRIPT]] uses <a>Unicode locales</a> as the basis for the APIs found in [[ECMA-402]].</p>
```

Comment:
This came from elsewhere? Also, it's a "forward definition" here, since it refers to international preferences before we introduce the term.

Reply:
Yep, it was previously in the language tags section, which it's not actually relevant to, and replaces something not as good in the locales section.
There are more than one forward definitions, but since they have links to the definition i'm not sure that's a problem.